.github/workflows/gradle.yml

```diff
@@ -18,8 +18,10 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java }}
-    - name: ./gradlew build javadoc
-      run: ./gradlew build
+    - name: Setup Environment
```

Comment:
```suggestion
    - name: ./gradlew build
      run: ./gradlew build
    - name: Setup local CF
```

Reply:
This keeps the build with the configured release and has the separate step for the local CF.